spec/requests/api/v1/npq_participants_spec.rb

```diff
@@ -129,6 +129,18 @@
         let(:params)            { { data: { attributes: { course_identifier: course_identifier, reason: Participants::Defer::NPQ.reasons.sample } } } }
         let(:withdrawal_params) { { data: { attributes: { course_identifier: course_identifier, reason: Participants::Withdraw::NPQ.reasons.sample } } } }
       end
+
+      it_behaves_like "JSON Participant Resume endpoint", "npq-participant" do
+        let(:course_identifier) { npq_course.identifier }
+        let(:url)               { "/api/v1/participants/npq/#{npq_application.user.id}/resume" }
+        let(:withdrawal_url)    { "/api/v1/participants/npq/#{npq_application.user.id}/withdraw" }
+        let(:params)            { { data: { attributes: { course_identifier: course_identifier } } } }
+        let(:withdrawal_params) { { data: { attributes: { course_identifier: course_identifier, reason: Participants::Withdraw::NPQ.reasons.sample } } } }
+        before do
+          put "/api/v1/participants/npq/#{npq_application.user.id}/defer",
```

Comment:
```suggestion
          put "/api/v1/participants/npq/#{npq_application.user.id}/resume",
```

Reply:
@jamesrose you do need to defer the application first

Reply:
Whoops, I misread that. Thanks.